src/hooks/types.ts

```diff
@@ -14,6 +14,6 @@ export interface RectangleOptions {
   style?: CSSProperties;
 }
 export interface PathOptions {
-  color?: CSSProperties['stroke'];
+  color?: CSSProperties['stroke' | 'color'];
```

Comment:
What does this do?

Reply:
i fixed it , it's suppoosed to be fill , because we have polygone or polyline